Fix up DBan a bit

end360 opened the pull request:
- Remove a 10 second timer that spammed console commands.
- Add a callback function rather than a 1 second timer for networking
- Switch to net library instead of usernemssages

end360 committed:
Fix up DBan a bit

- Remove a 10 second timer that spammed console commands.
- Add a callback function rather than a 1 second timer for networking
- Switch to net library instead of usernemssages

kralicky commented on CustomCommands/lua/autorun/client/cl_dban.lua:
```diff
@@ -123,99 +122,76 @@ function OpenPanelZZ4Z( ply, cmd, args, str )
-			local x = string.gsub( v[2], "%d", "-" )
-			list:AddLine( v[1], k, x, v[3] )
+			for k,v in ipairs( disconnectTable ) do
+				local x =
```
accidentally left this line in?

end360 replied:
Fixed. Slipped past me because it didn't end up causing any problems and the linter didn't pick it up.

kralicky commented on CustomCommands_onecategory/lua/autorun/client/cl_dban.lua:
```diff
@@ -123,99 +122,76 @@ function OpenPanelZZ4Z( ply, cmd, args, str )
-			local x = string.gsub( v[2], "%d", "-" )
-			list:AddLine( v[1], k, x, v[3] )
+			for k,v in ipairs( disconnectTable ) do
+				local x =
```
Same here

end360 replied:
Fixed. Slipped past me because it didn't end up causing any problems and the linter didn't pick it up.

end360 committed:
Copy and paste bad

Fix an issue with copy and pasting. Technically didn't cause problems because lua interpreted it as local x = list... but not good to make useless locals.

cobalt77 commented:
Thank you!
